src/components/Containers/Message/ParsedMessageBody.js

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styled from 'styled-components';
 import { colorPrimary } from '../../../styles/styleVariables';
 
+// Context
+import { MessageContext } from './MessageMain';
+
 const ParsedMessageBody = ({ message }) => {
+  const { viewAsPlaintext } = useContext(MessageContext);
   return (
     <MessageBodyWrapper>
       <MessageBodyStart>START MESSAGE BODY</MessageBodyStart>
-      <BodyContent dangerouslySetInnerHTML={{ __html: message.body }} />
+      {viewAsPlaintext ? (
+        <PlainBodyContent>{message.body}</PlainBodyContent>
```

Comment:
In other parts of the code aren't these built like this `<PlainBodyContentStyled>`. Is there are reason these don't follow that convention? Curious mostly.

Reply:
In truth, I haven't settled on a convention for nested styled-components. I think maybe what I like (but haven't done here) is something like this:
1. If the component is called Header, the outermost JSX tag should be called `<HeaderStyled>`.
2. Inner styled-components should be called `<HeaderNavList>` or `<HeaderListItem>`. Basically just namespace everythign inside the component.

That's something I've started trying mid-way through this project, so it hasn't really made its way in.